tools/test/examples/examples.json

```diff
@@ -469,9 +469,9 @@
         "github": "https://github.com/ARMmbed/mbed-os-example-atecc608a",
         "mbed": [],
         "test-repo-source": "github",
-        "features" : [],
+        "features" : ["HAS_CRYPTOKIT"],
```

Comment:
Does this "features" setting correlate at all with the RaaS tag? There is no HAS_CRYPTOKIT Mbed OS feature.